### ISSUE TITLE
Create a Physics Informed Neural Network that models a physical system

### ISSUE THREAD

**Issue by RoshenS** [RoshenS/PINNs#3]
This can be done by performing the following:

1. Select a physical system to model.
2. Sample from the system.
3. Incorporate the equations describing the physical system into the loss function in addition to the MSE of samples drawn as used in previous stages.
